blog/_posts/2021-08-20-Aria-Lanea-Blog.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: post
+title: "Planning for the Future of MABE2"
+date: 2021-08-20
+author: Aria Killibrew Bruehl and Lanea Rohan
+---
+
+## Outline
+
+# :soon: Planning for the Future of MABE2: A Summer of Documentation and Testing
+​
+Hello! This summer we (Lanea Rohan, '22 Grinnel College, and Aria Killebrew Bruehl,
+'23 Reed College) participated in the WAVES workshop through Michigan State University.
+We dedicated our summer to documenting and testing MABE2, an open-source research platform that provides accessible tools for conducting evolutionary computation and digital evolution research.
+​
+This project was guided by our wonderful mentors [Acaica Ackles](https://alackles.github.io/) (documentation) and [Austin Ferguson](http://fergusonaj.com/) (testing).
+
+​
+
+## :interrobang: Why Document and Test Code
+​
+The [Digital Evolution Lab](https://devolab.org/) has spent countless hours developing MABE2 and preparing for its release. Proper documentation and testing are essential for ensuring that MABE2 is accepted into and used by as large of a community as possible.
+​
+Write the Docs (link!), BLURB, puts the importance of proper documentation quite plainly:
+​
+> If people don’t know why your project exists,they won’t use it.
+> If people can’t figure out how to install your code, they won’t use it.
+> If people can’t figure out how to use your code, they won’t use it.
+​
+This is particularly important for the end-users of MABE2 who simply want to run experiments and do not want to waste time trying to understand the source code on their own, especially users who have less coding experience.
+​
```

Comment:
I love the 3 > points above, I think it's both really important and a good message. It goes hand in hand with our work on DynamicOrg,  making the end user experience as easy as possible. Good work!